libs/vkjson/vkjson.cc

```diff
@@ -34,11 +34,16 @@
 
 #include <cJSON.h>
 #include <vulkan/vk_sdk_platform.h>
+#include <stdlib.h>
 
 namespace {
 
 inline bool IsIntegral(double value) {
+#if defined(ANDROID)
+  return trunc(value) == value;
```

Comment:
The NDK's STL support is partial, trunc isn't included (although it might be in NDK r16).  This instead uses trunc from the NDK in sysroot/usr/include/math.h